frontend/components/utility/HighlightButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { MouseEventHandler } from "react";
```

Comment:
not too sure if `utility` is the best folder to put this under.. atleast it doesn't scream that to me initially when i first look at it.

Possibly, should we have a commons folder? and like stuff like input/ buttons/ dropdowns all do in there? im not too sure, what did your frontend course structure it as?